Documentation/gettingstarted/egress-gateway.rst

```diff
@@ -379,12 +385,36 @@ cluster with the IP of the node.
 Apply egress gateway policy
 ---------------------------
 
+Download the ``egress-sample`` Egress Gateway Policy yaml:
+
+.. code-block:: shell-session
+
+    $ wget \ |SCM_WEB|\/examples/kubernetes-egress-gateway/egress-nat-policy-egress-gateway.yaml
+
+Modify the ``destinationCIDRs`` to include the IP of the host where your
+designated external service is running on.
```

Comment:
This could be made into a single cmd by using kubectl patch. For example something like `kubectl patch deployment patch-demo -p '{"spec": {"replicas": 2}}'`. This can be improved as a follow up and is not a blocker for this PR to be merged.